core/include/mmio.h

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
If mmio.h and mmio.c are not related with VTLB code clean, better submit in another commit.

Reply:
Thanks for your careful review. These MMIO functions had already existed in VTLB module before. All functions will be invoked by vCPU module currently, so they cannot be omitted because of build reason. All changes this time are atomic. For more details, see the comments at line 727 in previous [vtlb.c](https://github.com/intel/haxm/blob/c06eb4ecc80549fbf9ddd5602260ef0e53dd8401/core/vtlb.c#L727).